checker/src/test/java/org/checkerframework/checker/test/junit/ResourceLeakInferenceTest.java

```diff
@@ -0,0 +1,26 @@
+package org.checkerframework.checker.test.junit;
+
+import java.io.File;
+import java.util.List;
+import org.checkerframework.checker.resourceleak.ResourceLeakChecker;
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+/** Tests for the Resource Leak Checker. */
+public class ResourceLeakInferenceTest extends CheckerFrameworkPerDirectoryTest {
```

Comment:
I am still unclear on how automated testing should work; can you clarify?  I think you said there is a system test to make sure the inference is working.  If so, we should delete this class and `CheckFields.java`, and then update the relevant WPI testing scripts so that the new system test is always run.  There should be some change in this PR that adds a new system test

Reply:
https://github.com/typetools/checker-framework/pull/4926/commits/9449840ffca5cf19c1c5ce46a9263310f5956537